app/models/external_activity.rb

```diff
@@ -3,6 +3,7 @@ class ExternalActivity < ActiveRecord::Base
 
   # see https://github.com/sunspot/sunspot/blob/master/README.md
   searchable do
+    integer :id
```

Comment:
I think it will work to do:
without(material)

I don't see it clearly in the docs. But I see a hint of it here: https://outoftime.lighthouseapp.com/projects/20339/tickets/72-adding-a-without-restriction-knowing-an-id-of-an-object
And also in the code: https://github.com/sunspot/sunspot/blob/master/sunspot/lib/sunspot/dsl/scope.rb#L207